### PR TITLE
replace /en/ to /en-US/docs/ in Web/API

### PR DESCRIPTION
**Pull request by peterbe**

### CODE REVIEW

**Comment by peterbe** [comment]
Weird! My updates didn't take effect!
```
▶ git push peterbe replace-en-to-en-usdocs-in-webapi
Everything up-to-date

```
There *are* more than 1 commit on this branch and I've pushed it to my fork, but it's not showing up here in GitHub. Must be a bug.

**Comment by peterbe** [comment]
Weird! My updates didn't take effect!
```
▶ git push peterbe replace-en-to-en-usdocs-in-webapi
Everything up-to-date

```
There *are* more than 1 commit on this branch and I've pushed it to my fork, but it's not showing up here in GitHub. Must be a bug.

File: files/en-us/web/api/css_object_model/using_dynamic_styling_information/index.html [hunk 1]
```diff
@@ -91,11 +91,19 @@ <h2 id="Modify_an_element_style">Modify an element's style</h2>
 
 <p>{{ EmbedLiveSample('Modify_an_element_style') }}</p>
 
+<<<<<<< HEAD
```
Comment (peterbe) [inline files/en-us/web/api/css_object_model/using_dynamic_styling_information/index.html:94]: @nschonni @bershanskiy To err is human. But for CI not to catch it is a failure. 
I must have fumbled when I fixed the merge conflicts. 
I'll file a new issue about this.

Comment (nschonni) [reply files/en-us/web/api/css_object_model/using_dynamic_styling_information/index.html]: VSCode yells at you when you try to stage files with conflicts during a rebase 😉

**Review by jpmedley — APPROVED** [review]

**Pull request merged**